{{cookiecutter.project_dirname}}/.env_template

```diff
@@ -1,16 +1,17 @@
 CACHE_URL=locmem://
 COMPOSE_FILE=docker-compose.yaml
-DATABASE_URL=postgres://postgres:postgres@postgres:5432/{{cookiecutter.project_slug}}
-DJANGO_ADMINS=admin,errors@{{cookiecutter.domain_url}}
-DJANGO_ALLOWED_HOSTS=localhost
+DATABASE_URL=postgres://postgres:postgres@postgres:5432/{{ cookiecutter.project_slug }}
+DJANGO_ADMINS=admin,errors@example.org
+DJANGO_ALLOWED_HOSTS=localhost,{{ cookiecutter.service_slug }}
+DJANGO_BUILD_TARGET=local
 DJANGO_CONFIGURATION=Local
 DJANGO_DEBUG=True
-DJANGO_DEFAULT_FROM_EMAIL=info@{{cookiecutter.domain_url}}
+DJANGO_DEFAULT_FROM_EMAIL=info@example.org
```

Comment:
the placeholder {{ cookiecutter.domain_url }} could still be useful.

Reply:
No need for that in local development.